### ISSUE TITLE
Swing error prompt editable

### ISSUE THREAD

**Issue by HFellows** [Clojure-Intro-Course/clojure-intro-class#63]
It seems like poor practice to make the swing prompt editable. It implies that users can interact with it and expect results.

**Comment by elenam** [comment]
True, but I wouldn't worry about this now. All the display stuff is preliminary.
